packages/less/benchmark/benchmark-runner.js

```
@@ -94,6 +97,25 @@ function hrNow() {
  * @returns {void}
  */
 function runOnce(callback) {
+    var start = hrNow();
+    var opts = {
+        filename: filePath,
+        paths: [fileDir]
+    };
+    // Forward extra options (e.g. --math=always)
+    for (var key in extraOpts) { opts[key] = extraOpts[key]; }
+    less.render(data, opts, function (err, output) {
```

ESLint

[error] 'output' is defined but never used.

(no-unused-vars)